### ISSUE TITLE
De-Twitterfication of dashboard?

### ISSUE THREAD

**Issue by mellowmaromi** [AprilSylph/XKit-Rewritten#1436]
### Concept

Now that Dashboard Unfucker officially doesn't work anymore, is there any way to restore the layout of the dashboard to the classic tumblr version instead of the three column twitter ripoff?

**Comment by krc** [comment]
I just posted in discussion about this... it really sucks. If xkit could get this, or if anyone knows an alternate script or extension, that would be great...

**Comment by librasuns** [comment]
PLEASE!!! Now without Dashboard Unfucker, the dashboard is really unusable and un user-friendly. Even having the option of having your likes, drafts, queue and secondary blogs visible on the dash without having to go into account (hide follower count doesn't work anymore either) that would be great.

**Comment by krc** [comment]
I still hope xkit might integrate a fix, but in the meantime,  some people on tumblr point me to this userstyle 

https://www.tumblr.com/pixiel/723402150351437824/anyway-i-created-a-stylish-stylus-that

https://userstyles.world/style/11286/old-tumblr-dashboard-userstyle

**Comment by mellowmaromi** [comment]
I hope xkit implements a stable fix (obviously, why else would I have posted this), but thank you so much for those links!

**Comment by librasuns** [comment]
omg, thank you so much!!!

**Comment by marcustyphoon** [comment]
I'm certainly not devoting my time to maintaining that, no. I won't deign to speak for April, but I can't imagine she will either. The user style linked seems to work at the moment, and if someone wants help making something of their own I'm not against supplying that.

Might be worth leaving this open and wontfix so people see it.

**Comment by librasuns** [comment]
That's understandable. 

I'm thankful for you two for XKit and the user who supplied the userstyle substitute for dashboard un-fucker. Y'all already have a lot on your plate so it wouldn't be fair to expect y'all to carry yet another load to make Tumblr usable. 

Hopefully the user style stays workable long term.

**Comment by marcustyphoon** [comment]
Eh, I mean, I think it's more just that I don't use old layout modifications. It's a lot more work maintaining something you don't use, as that entails relying on reports from others (often with insufficient information) or turning it on regularly to know what actually needs fixing.

**Comment by nightpool** [comment]
How do people feel about something like? 
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/233815/4ead9a2e-54cb-4492-a95f-6c8c5ca5fc17)

**Comment by librasuns** [comment]
It feels a little less cluttered to what Tumblr has now, but personally speaking I'd still feel claustrophobic. 

I'd be okay using you iteration if there was an easy way to access likes, drafts, and queue.

**Comment by mellowmaromi** [comment]
Agreed; it's better than tumblr's current model, but I think the biggest thing that's weirding me out is the leftmost column, since I'm used to all those buttons being at the top of the screen.

**Comment by nightpool** [comment]
@mellowmaromi Unfortunately I don't think that adding a new sticky top nav bar is in the cards for us—it's just a really big deviation from the way the site currently works and it would be really difficult to maintain in the long term. I think I'm coming around to the idea of a "collapsible" left sidebar where the buttons get rolled up into a hamburger menu or something similar, but I don't think a top navbar is something the rest of the devs would go for (and it would be a fair amount of work to implement)

**Comment by mellowmaromi** [comment]
Yeah, I kind of figured that's the way things are going... Tumblr seems really dead set on getting rid of the sticky top nav bar no matter what, for whatever reason. Still really appreciate all the work you and the other devs are doing on all the other features, though!

**Comment by AprilSylph** [comment]
I see no reason to compete with the Old Tumblr Dashboard userstyle on this front.
